app/views/super_admin/super_admin.py

```diff
@@ -35,6 +36,7 @@ def index_view():
     rejected_sessions = get_count(DataGetter.get_all_rejected_sessions())
     draft_sessions = get_count(DataGetter.get_all_draft_sessions())
     email_times = DataGetter.get_email_by_times()
+    total_users= super_admins + admins + registered_users + unverified_users
```

Comment:
This would give a wrong count ... An admin by default is a registered user ... So, the count would be more than actual